docker/Dockerfile

```diff
@@ -10,7 +10,7 @@ ARG xla_branch=""
 ARG example_branch="master"
 ARG cuda="0"
 ARG cuda_compute="3.7,7.0,7.5,8.0"
-ARG cxx_abi="1"
+ARG cxx_abi="0"
```

Comment:
this only affect docker/wheel build, not CI. If you want to test this, you want to trigger the docker/wheel build targeting this pr.

Reply:
Thanks. Very fast review. Let me paste the result here once I get it.

Reply:
Will monitor the [log](https://pantheon.corp.google.com/cloud-build/builds;region=us-central1/85997a34-6aaa-43a4-9bcc-0c5f69f00a98?e=13802955&jsmode=o&mods=allow_workbench_image_override&project=tpu-pytorch)

Reply:
Failed to install torchvision:
```
Step #0: [0m[91m+ git clone -b main https://github.com/pytorch/vision.git
Step #0: [0m[91mCloning into 'vision'...
Step #0: [0m[91m+ pushd vision
Step #0: [0m/pytorch/vision /pytorch
Step #0: [91m+ python setup.py bdist_wheel
Step #0: [0m[91mTraceback (most recent call last):
Step #0:   File "setup.py", line 68, in <module>
Step #0:     pillow_req = "pillow-simd" if get_dist("pillow-simd") is not None else "pillow"
Step #0:   File "setup.py", line 22, in get_dist
Step #0:     return get_distribution(pkgname)
Step #0:   File "/root/anaconda3/envs/pytorch/lib/python3.8/site-packages/pkg_resources/__init__.py", line 514, in get_distribution
Step #0:     dist = get_provider(dist)
Step #0:   File "/root/anaconda3/envs/pytorch/lib/python3.8/site-packages/pkg_resources/__init__.py", line 386, in get_provider
Step #0:     return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
Step #0:   File "/root/anaconda3/envs/pytorch/lib/python3.8/site-packages/pkg_resources/__init__.py", line 956, in require
Step #0:     needed = self.resolve(parse_requirements(requirements))
Step #0:   File "/root/anaconda3/envs/pytorch/lib/python3.8/site-packages/pkg_resources/__init__.py", line 815, in resolve
Step #0:     dist = self._resolve_dist(
Step #0:   File "/root/anaconda3/envs/pytorch/lib/python3.8/site-packages/pkg_resources/__init__.py", line 844, in _resolve_dist
Step #0:     env = Environment(self.entries)
Step #0:   File "/root/anaconda3/envs/pytorch/lib/python3.8/site-packages/pkg_resources/__init__.py", line 1044, in __init__
Step #0:     self.scan(search_path)
Step #0:   File "/root/anaconda3/envs/pytorch/lib/python3.8/site-packages/pkg_resources/__init__.py", line 1077, in scan
Step #0:     self.add(dist)
Step #0:   File "/root/anaconda3/envs/pytorch/lib/python3.8/site-packages/pkg_resources/__init__.py", line 1096, in add
Step #0:     dists.sort(key=operator.attrgetter('hashcmp'), reverse=True)
Step #0:   File "/root/anaconda3/envs/pytorch/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2631, in hashcmp
Step #0:     self.parsed_version,
Step #0:   File "/root/anaconda3/envs/pytorch/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2678, in parsed_version
Step #0:     self._parsed_version = parse_version(self.version)
Step #0:   File "/root/anaconda3/envs/pytorch/lib/python3.8/site-packages/pkg_resources/_vendor/packaging/version.py", line 266, in __init__
Step #0:     raise InvalidVersion(f"Invalid version: '{version}'")
Step #0: pkg_resources.extern.packaging.version.InvalidVersion: Invalid version: '4.0.0-unsupported'
Step #0: The command '/bin/sh -c cd /pytorch/xla && bash scripts/build_torch_wheels.sh ${python_version} ${release_version} ${build_cpp_tests}' returned a non-zero code: 1
Finished Step #0
ERROR
ERROR: build step 0 "gcr.io/cloud-builders/docker" failed: step exited with non-zero status: 1
```
Let me try a different way of installing torchvision.

Reply:
known issue, check https://github.com/pytorch/xla/pull/4906

Reply:
btw, if you check the log, you might need to do what I do in 
https://github.com/pytorch/xla/pull/4906/files#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddcR60

to make trigger actually pick up the docker file specified by you. Check the log to confirm that ABI change is taking affect in the log.

Reply:
Good finding. I guess you found it [here](https://github.com/pypa/setuptools/issues/3772)?

The [build](https://pantheon.corp.google.com/cloud-build/builds;region=us-central1/4101bc2f-a633-4a74-a336-08f1e460f6c3?e=13802955&jsmode=o&mods=allow_workbench_image_override&project=tpu-pytorch) seems good. TPU CI is running (I guess you started it? Thanks for that.)